Gems/EMotionFX/Code/Include/Integration/ApplyMotionSetComponentBus.h

```diff
@@ -0,0 +1,61 @@
+#pragma once

```

Comment:
Validator fails for all files H and CPP in your PR
with message
**Validation / build (pull_request)**

Reply:
Alexander O. fixed this issue ^^^ for his PR in O3DE fork some time ago.

Reply:
These are tabs. The validator dislikes them. Deleted.